### PR TITLE
Dfe zendesk

### PR DESCRIPTION
**Pull request by Marianie-Alphonse**
Remove the Manage Ontologies and Add Ontologies pages from the UI

### CODE REVIEW

**Review by Marianie-Alphonse — COMMENTED** [review]
Should I remove config.json?

File: public/config.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "apiUrl": "https://consent.dsde-dev.broadinstitute.org/api",
+  "ontologyApiUrl": "https://consent-ontology.dsde-dev.broadinstitute.org/",
+  "clientId": "1087760099392-u18o6sqvb8uljmdltflohmd34h6hik9b.apps.googleusercontent.com",
+  "firecloudUrl": "https://firecloud-orchestration.dsde-dev.broadinstitute.org/",
+  "gwasUrl": "",
+  "nihUrl": "http://mock-nih.dev.test.firecloud.org/link-nih-account/index.html"
+}
```
Comment (rushtong) [inline public/config.json:8]: This file should be removed, it's in `.gitignore`

File: src/libs/models.js [hunk 2]
```diff
@@ -40,7 +40,6 @@ export const Models = {
       population: false,
       other: false,
       otherText: "",
-      ontologies: [],
```
Comment (rushtong) [inline src/libs/models.js:43]: We still need this, it's not part of the add/remove ontologies feature.

File: src/pages/DataAccessRequestApplication.js [hunk 3]
```diff
@@ -54,7 +54,6 @@ class DataAccessRequestApplication extends Component {
         hmb: false,
         poa: false,
         diseases: false,
-        ontologies: [],
```
Comment (rushtong) [inline src/pages/DataAccessRequestApplication.js:57]: Revert changes to this file, the ontologies referred to here is not the same thing as adding and removing ontology files in GCS, this is for ontology terms that are part of a data access request.

File: src/pages/DataAccessRequestRenewal.js [hunk 4]
```diff
@@ -54,7 +54,6 @@ class DataAccessRequestRenewal extends Component {
         hmb: false,
         poa: false,
         diseases: false,
-        ontologies: [],
```
Comment (rushtong) [inline src/pages/DataAccessRequestRenewal.js:57]: Same feedback here as for `DataAccessRequestApplication.js`

File: src/pages/DatasetRegistration.js [hunk 5]
```diff
@@ -49,7 +49,6 @@ class DatasetRegistration extends Component {
         hmb: false,
         poa: false,
         diseases: false,
-        ontologies: [],
```
Comment (rushtong) [inline src/pages/DatasetRegistration.js:52]: Same feedback here as for `DataAccessRequestApplication.js`

File: src/pages/dar_application/TypeOfResearch.js [hunk 6]
```diff
@@ -7,22 +7,6 @@ import * as fp from 'lodash/fp';
 
 export const TypeOfResearch = hh(class TypeOfResearch extends Component {
 
-  searchOntologies = (query, callback) => {
```
Comment (rushtong) [inline src/pages/dar_application/TypeOfResearch.js:10]: Same feedback here as for `DataAccessRequestApplication.js`

**Review by rushtong — COMMENTED** [review]
See feedback inline, looks good but we've pruned a little too much from the tree! 
Also, there are now likely unused API calls (the ones to upload/download/delete ontology GCS files) in `ajax.js` that we no longer need as well.

**Comment by rushtong** [comment]
One more bit of feedback. Can we name this PR with a ticket number and little more descriptive? The `zendesk` title is confusing. Thank you!

**Comment by Marianie-Alphonse** [comment]
Thank you for the comments! I realize too late that I should have made a new fresh temp branch, then use an old one. I'm going to make one and add the right changes.

**Comment by rushtong** [comment]
Sounds good, thank you!

**Pull request closed without merging**